docs/reference/faq-troubleshooting.md

```diff
@@ -1,17 +1,75 @@
 # FAQ & Troubleshooting
 
-## What can I do about `error parsing torrent at http://…`?
+### What can I do about `error parsing torrent at http://…`?
 
 This means that the jacket download link didn't resolve to a torrent file. It's
 possible you got rate-limited so you might want to try again in a day or more.
 Otherwise, just ignore it. There's nothing cross-seed will be able to do to fix
 it.
 
-## rtorrent injected torrents don't check (or start at all) until force rechecked
+### rtorrent injected torrents don't check (or start at all) until force rechecked
+
 Remove any `stop_untied=` schedules from your .rtorrent.rc.
 
 Commonly: `schedule2 = untied_directory, 5, 5, (cat,"stop_untied=",(cfg.watch),"*.torrent")`
 
-## Failed to inject, saving instead.
+### Failed to inject, saving instead.
+
+Best way to start troubleshooting this is to check the `verbose.\*.log` and find this specific event.
+
+You will be able to see the circumstances around the failure, and start investigating why this occurred.
+
+### My data-based results are paused at 100% after injection
+
+This is by design and due to the way data-based searches function. This is done to prevent downloading missing files
+or files that failed rechecking automatically. Until there is a way to guarentee you won't end up downloading
+extra (not cross-seeding) this is the best solution.
+
+Consider using infoHash if you are racing using data-based searches to prevent mismatches.
```

Comment:
infoHash uses the .torrent matching pipeline, not data-based. I like the recommendation for infoHash if they're racing but the second half of the sentence should go

Reply:
Yea I don't know where that came from, I wrote it at 4AM....so who knows.